### ISSUE TITLE
"fresch db": Consider adding support for specifying specific tables to sync/ignore

### ISSUE THREAD

**Issue by danielkalen** [mspae/fresch#1]
e.g.
`mysqldump -u XXXX -p XXXX --ignore-table="table1"  --ignore-table="table2"`

**Comment by mspae** [comment]
Good idea, I'm open for PRs for this feature. Unfortunately I don't have time to implement it myself at the moment.
